test/vvenc_unit_test/vvenc_unit_test.cpp

```diff
@@ -345,6 +347,75 @@ static bool check_cpyCoeff( TCoeffOps* ref, TCoeffOps* opt, unsigned num_cases )
   return passed;
 }
 
+static bool check_roundClip( TCoeffOps* ref, TCoeffOps* opt, unsigned num_cases )
+{
+  DimensionGenerator dim;
+
+  static constexpr size_t buf_size = MAX_CU_SIZE * MAX_CU_SIZE;
+
+  // Use xMalloc to create aligned buffers for x86.
+  TCoeff *dst_ref = ( TCoeff* )xMalloc( TCoeff, buf_size );
+  TCoeff *dst_opt = ( TCoeff* )xMalloc( TCoeff, buf_size );
+
+  bool passed = true;
+
+  // Use input values up to signed 22-bit based from a real encode.
+  InputGenerator<TCoeff> inp_gen{ 22, /*is_signed=*/true };
+
+  std::ostringstream sstm_test;
+  sstm_test << "TCoeffOps::roundClip";
+  std::cout << "Testing " << sstm_test.str() << std::endl;
+
+  for( unsigned n = 0; n < num_cases; n++ )
+  {
+    // Set height, width, and stride to powers of two.
+    unsigned log2height = dim.get( 0, 7 );
+    unsigned log2width = dim.get( 2, 7 );
```

Comment:
Strictly speaking, the largest transform block in VVC is 64x64, so both W/H should only go until 64.